magento2/usr/local/share/magento2/magento_functions.sh

```diff
@@ -386,6 +386,10 @@ function do_magento_download_magerun2() {
 }
 
 function do_magento2_templating() {
+  if [ "$MAGENTO_VERSION" != "2.0" ] && [ "$MAGENTO_VERSION" != "2.1" ]; then
```

Comment:
there's also composer_package_compare that can be used here if you can always match a package (for CE and EE)

Reply:
(or could deconstruct the comparison part)